packages/patternfly-4/gatsby-theme-patternfly-org/components/footer/footer.css

```diff
@@ -99,6 +99,8 @@
 .ws-org-pfsite-l-footer-dark {
   padding: 1rem !important;
   background: #151515 !important;
+  /* disables the default flex-grow: 1 for the last .pf-c-page__main-section in a block */
+  flex-grow: 0;
```

Comment:
We need to increase the specificity of the selector that applies this style, since the original style's specificity is 20 (`.pf-c-page__main-section:last-of-type`). In [my other comment](https://github.com/patternfly/patternfly-org/pull/1699#issuecomment-590454683) I recommended `.pf-c-page__main-section.ws-org-pfsite-l-footer-dark`, but you could also use `.ws-org-pfsite-l-footer-dark:last-child` or `.ws-side-nav-layout .ws-org-pfsite-l-footer-dark`. And I noticed you use `!important` in these styles, that would also work here, though I generally try to avoid `!important` but that's just me :)

This is the way it looks now

<img width="1221" alt="Screen Shot 2020-02-27 at 2 55 23 PM" src="https://user-images.githubusercontent.com/35148959/75486319-e49ea480-5971-11ea-8201-5b8a0ca64980.png">

After the change, it should look like this (the main section fills all of the available space)

<img width="1221" alt="Screen Shot 2020-02-27 at 2 55 08 PM" src="https://user-images.githubusercontent.com/35148959/75486339-f41ded80-5971-11ea-8e87-f3427d42d9ab.png">

Reply:
Updated this - I had to combine your original selector `.pf-c-page__main-section.ws-org-pfsite-l-footer-dark` and add the parent selector `.ws-side-nav-layout` to override the existing flex styling.